lib/rucio/web/rest/flaskapi/v1/heartbeats.py

```diff
@@ -49,12 +51,41 @@ def get(self):
         """
         return Response(json.dumps(list_heartbeats(issuer=request.environ.get('issuer'), vo=request.environ.get('vo')), cls=APIEncoder), content_type='application/json')
 
+    def post(self):
+        """
+        Accepts a heartbeat.
+
+        .. :quickref: Heartbeat; Accepts a heartbeat.
+
+
+        :<json dict parameter: Dictionary with 'executable', 'hostname', 'pid', 'thread', 'older_than', 'payload'
+        :status 201: Created.
+        :status 400: Cannot decode json parameter list.
+        :status 401: Invalid Auth Token.
```

Comment:
I don't see the 401 error generated in the code below. Neither do I see the 201 created. 

See https://github.com/rucio/rucio/blob/02e86830f10c1bd7e5ea2a05d306706a09127bfe/lib/rucio/web/rest/flaskapi/v1/vos.py#L152 for example

Reply:
true. now fixed.